pymc3/distributions/discrete.py

```diff
@@ -141,11 +147,14 @@ class Bernoulli(Discrete):
     p : float
         Probability of success (0 < p < 1).
     """
-    def __init__(self, p, *args, **kwargs):
-        super(Bernoulli, self).__init__(*args, **kwargs)
-        self.p = p
+    def __init__(self, p, ndim=None, size=None, dtype=None, *args, **kwargs):
+        self.p = T.as_tensor_variable(p)
```

Comment:
I meant `T.as_tensor_variable` and your reasoning makes sense, good clean living.